cjwstate/modules/i18n/__init__.py

```diff
@@ -0,0 +1 @@
+from cjwstate.modules.i18n.catalogs import update_module_catalogs
```

Comment:
To me, it seems like `update_module_catalogs()` does for modules what `bin/dev extract-messages` does for our core code. Should this be called `extract_module_messages()`? (Is the concept different enough to warrant a new verb?)